client/src/App.js

```diff
@@ -15,6 +16,7 @@ function App() {
         <Navbar />
         <Route exact path="/" component={LandingPage} />
         <Route exact path="/me" component={LandingPage} />
+		<Route exact path="/profile/edit" component={ProfileEdit} />
```

Comment:
would be good to follow convention of #26 and have the url be /edit-profile

Reply:
To add on to that, I had built that skeleton around storing the components within the "/me" component